src/components/AgreementLineSections/FormPOLines/FormPOLines.test.js

```diff
@@ -0,0 +1,247 @@
+import React from 'react';
+import '@folio/stripes-erm-components/test/jest/__mock__';
+import {
+  TestForm,
+  renderWithIntl,
+} from '@folio/stripes-erm-components/test/jest/helpers';
+import { Accordion } from '@folio/stripes-testing';
+import FormPOLines from './FormPOLines';
+import translationsProperties from '../../../../test/helpers';
+
+jest.mock('../../POLinesFieldArray', () => () => <div>POLinesFieldArray</div>);
+const onSubmit = jest.fn();
+
+const formPolines = {
+  'addButtonTooltipId': 'ui-agreements.agreementLine.addCustomCoverageTootlip',
+  'agreementLineSource': 'basket',
+  'basket': [],
+  'isSuppressFromDiscoveryEnabled': 'ƒ () {}',
```

Comment:
Please make sure we have proper functions being passed instead of strings , i.e
```suggestion
  'isSuppressFromDiscoveryEnabled': () => {},
```